app/views/user_mentee_applications/index.html.erb

```diff
@@ -15,12 +15,14 @@
     <% end %>
 
 
-    <h2 class="text-xl font-bold mb-2">Past Applications</h2>
-    <ul>
-      <% @user_mentee_applications.each do |mentee_application| %>
-        <% next if mentee_application.active? && mentee_application.in_review? %>
-        <%= render UserMenteeApplication::ListItemComponent.new(mentee_application:) %>
-      <% end %>
-    </ul>
+    <% if @user_mentee_applications.present? %>
```

Comment:
this still shows up incorrectly, as `@most_recent_application` is a subset of `@user_mentee_applications`, which means that keying in on @user_mentee_applications continues to show up.
Instead of pushing more logic into the view, I would recommend updating how `@user_mentee_applications` is assigned.

Reply:
sure, let me get back to you on this after.

Reply:
Hey @afogel - Sorry I'm a little confused here :sweat_smile:. 
- I believe it's a View's logic to display the heading (_Past Applications_) conditionally. 
- How this logic can be moved to somewhere else?
- Moreover, `@most_recent_application` empty state logic is already being handled in view as well. 
- I am confused there this can go wrong. :worried: , Please guide me. Thanks

Reply:
sure -- so what you had is definitely part of the solution, but the hiccup comes when you're testing one an applicant who has already submitted an application (see your seed file for an example). 
In that case, they should have a submitted application and no past applications, but the "Past Applications" header still shows up.
So, without altering the existing view logic that you've introduced here, how can we make the issue go away?

Reply:
Ohh, got it..Let me fix thatl. Thanks for the guide.

Reply:
@afogel - I tried to optimize the solution by adding a scope for past applications [here](https://github.com/agency-of-learning/PairApp/pull/365/commits/3a8c2ce274b2b0b92175a50aec87ec02e9976f18). Please review.